### DISCUSSION TITLE
Targetting an entire array with dirty states

### DISCUSSION THREAD

**Discussion by svenadlung** [livewire/livewire#4044] [3. Feature ideas]
Not sure if this is a bug or a feature request though 😬

The loading states allow to trigger an indicator on any change of a property in an array by targetting the entire array (as described here: https://laravel-livewire.com/docs/2.x/loading-states#targeting-actions):

```
<div>
    <input type="text" wire:model="post.title">
    <input type="text" wire:model="post.author">
    <input type="text" wire:model="post.content">
 
    <div wire:loading wire:target="post">
        Updating Post...
    </div>
</div>
```

The same would be useful for dirty states but when I try to use 

```
<div>
    <input type="text" wire:model="post.title">
    <input type="text" wire:model="post.author">
    <input type="text" wire:model="post.content">
 
    <div wire:dirty wire:target="post">
        The form is dirty...
    </div>
</div>
```

it doesn't trigger anything.

In our sepcial usecase we want to enable a submit button only once there is at least one form field dirty.

Any idea?

**Comment by joshhanley** [comment]
@svenadlung I added the `wire:target` to an entiry array feature for `wire:loading` but I wasn't aware `wire:dirty` also uses it. So it's not a bug, in so much as it was never built to do that haha.

But PR's are welcome if you want to add that in 🙂 I have moved this to a feature request.

**Reply by svenadlung** [reply]
@joshhanley I already thought so. Thank you for your efforts.

**Reply by joshhanley** [reply]
No worries 🙂
